app/views/request/_sidebar.html.erb

```diff
@@ -15,44 +15,25 @@
     <%= render :partial => 'track/rss_feed', :locals => { :track_thing => @track_thing, :location => 'sidebar' } %>
   </div>
 
-  <% if @info_request.described_state != "attention_requested" %>
```

Comment:
I think we need to reconsider what we do with this section. It's already a bit weird:

<table>
  <tr>
    <th>Normal State</th>
    <th>Reported State</th>
  </tr>
  <tr>
    <td>
      <img src="https://cloud.githubusercontent.com/assets/282788/18203087/655f9f68-710c-11e6-88a2-8622928504f1.png" />
     </td>
      <td>
        <img src="https://cloud.githubusercontent.com/assets/282788/18203130/c1ef2f32-710c-11e6-8e41-8913b3754a79.png" />
      </td>
    </tr>
</table>

There are a few different states depending on whether the request is hidden to some degree.

Do these make sense when they're detached from the "report" button?